jobs/sink-clustered-deployment.yml

```diff
@@ -46,20 +46,22 @@
         days-to-keep: 7
         artifacts-days-to-keep: 7
 
-    triggers:
-    - timed: "H 2 * * *"
-    - github-pull-request:
-        trigger-phrase: '/(re)?test ((all)|(centos-ci/sink-clustered/mini-k8s-({k8s_version})?))'
-        admin-list:
-          - obnoxxx
-          - phlogistonjohn
-          - gd
-          - spuiuk
-          - raghavendra-talur
-          - synarete
-          - anoopcs9
-        cron: H/5 * * * *
-        status-context: 'centos-ci/sink-clustered/mini-k8s-{k8s_version}'
+    triggers: !j2-yaml: |
```

Comment:
I'm not familiar with this syntax. What's this mean?

Reply:
_sink-clustered-deployment.yml_ is the only job file which when deployed renders 3 different jobs based on  _{k8s_version}_. Now in order to bring in some conditionals we need Jinja2 syntax for which [Inclusion tags](https://jenkins-job-builder.readthedocs.io/en/latest/definition.html#inclusion-tags) are used. Thus github pull request trigger can be conditionally added when _{k8s_version}_ is `latest`.